src/app-nordigen/app-nordigen.js

```diff
@@ -199,6 +200,16 @@ app.post(
             error_code: 'NORDIGEN_ERROR',
           });
           break;
+        case isAxiosError(error):
```

Comment:
Fun fact! This doesn’t actually narrow the `error` type away from `any`. Maybe https://github.com/microsoft/TypeScript/pull/53681 will fix that?